README.md

```diff
@@ -6,6 +6,7 @@
 [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/pfnet/optuna)
 [![CircleCI](https://circleci.com/gh/pfnet/optuna.svg?style=svg)](https://circleci.com/gh/pfnet/optuna)
 [![Read the Docs](https://readthedocs.org/projects/optuna/badge/?version=stable)](https://optuna.readthedocs.io/en/stable/)
+[![Codecov](https://codecov.io/gh/pfnet/optuna/branch/master/graph/badge.svg)](https://codecov.io/gh/pfnet/optuna/branch/master)
```

Comment:
Do you plan to update this badge manually? I think we need to add `codecov` job to `daily-build` workflow in `.circleci/config.yml` if we want to update it automatically.

Reply:
I've misunderstood the timing of `commit-build` of master branch. It will be executed when a new PR is merged into master. So, the badge will keep updated. I'm sorry for bothering you.